src/Dfe.PlanTech.Web/Views/Pages/Page.cshtml

```diff
@@ -34,5 +33,10 @@
         }
     }
     
+    if(Model.Page.Slug == "self-assessment")
```

Comment:
I'd like to try and get away from hardcoding stuff about specific pages in places. Is there any way to move this to the content model?

Reply:
Yeah mate this is good. The other solution would be to have a content model that is essentially blank, but something that just says "PUT SCHOOL NAME HERE". Problem with this is that we have a "Title" in each Page that always goes first; we'd probably have to refactor this into its own separate content model. Which is a lot more work and probably not worth currently!

So yes... this is very good IMO good job!